packages/ERTP/test/unitTests/test-amount-key-arith.js

```diff
@@ -0,0 +1,22 @@
+import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
+import { fc } from '@fast-check/ava';
+import { keyEQ, keyGTE } from '@agoric/store';
+
+import { AmountMath as m } from '../../src/index.js';
+import { arbAmount } from '../../tools/arb-amount.js';
+
+test('amount equality iff key equality', async t => {
+  await fc.assert(
+    fc.property(fc.record({ x: arbAmount, y: arbAmount }), ({ x, y }) => {
+      return t.true(m.isEqual(x, y) === keyEQ(x, y));
+    }),
+  );
+});
+
+test('amount >= iff key >=', async t => {
+  await fc.assert(
+    fc.property(fc.record({ x: arbAmount, y: arbAmount }), ({ x, y }) => {
+      return t.true(m.isGTE(x, y) === keyGTE(x, y));
+    }),
+  );
+});
```

Comment:
Actually, the premise of this PR is wrong, and these tests need to be revised to reflect that.

We currently have four kinds of Amounts:
   * nats
   * sets represented as arrays -- deprecated but widely used
   * copySets, which should be used instead of arrays representing sets
   * copyBags, for semi-fungibles

The premise of this PR is true for nats, copySets, and copyBags. But it is not true for deprecated arrays representing sets, so we need to exclude them from these tests.

(Separately, we will be deprecating copySets as well, switching all uses of arrays-representing-sets and copySets to copyBags. But this is not a concern of this PR.)